waspc/data/Generator/templates/react-app/src/queries/_query.js

```diff
@@ -1,16 +1,7 @@
 {{={= =}=}}
-import { callOperation } from '../operations'
-import * as resources from '../operations/resources'
-
-function {= queryFnName =}(args) {
-  return callOperation('{= queryRoute =}', args)
-}
-
-const queryCacheKey = '{= queryRoute =}'
-{= queryFnName =}.queryCacheKey = queryCacheKey
-
-const entitiesUsed = {=& entitiesArray =}
-resources.addResourcesUsedByQuery(queryCacheKey, entitiesUsed)
-
-export default {= queryFnName =}
+import { createQuery } from './core'
 
+export default createQuery(
+  '{= queryRoute =}',
+  {=& entitiesArray =},
+)
```

Comment:
Ideally, I would change the default export for a named one (for reasons described in #546), like this:
```javascript
export {= queryFnName =} = createQuery(
  '{= queryRoute =}',
  {=& entitiesArray =},
)
```
However, this would call for changing all our examples and updating the docs so I decided to leave the default export for now.